### PR TITLE
[ECO-1697] Add the `charting_library` git submodule

### PR DESCRIPTION
**Pull request by xbtmatt**
# Description

- [x] Create the `static` folder in the repo and update the `.gitmodules` file
- [x] Update `inbox` `.gitmodules` to use `https` instead of `ssh`
- [x] Update this repo's `.gitmodules` in the same way
- [x] Update the `README.md` in `inbox`
- [x] Update the `README.md` in this repo
- [x] Add `tradingview` to the cspell dictionary

# Checklist

- [x] Did you update relevant documentation (if applicable)?
- [x] Did you add tests to cover new code or a fixed issue (if applicable)?
- [x] Did you update the changelog (if applicable)?
- [x] Did you check all checkboxes from the linked Linear task (if applicable)?
  (Ignore if you are not a member of Econia Labs)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,4 +3,4 @@
 	url = git@github.com:econia-labs/inbox.git
 [submodule "src/typescript/frontend/public/static"]
 	path = src/typescript/frontend/public/static
-	url = https://github.com/tradingview/charting_library.git
+	url = git@github.com:tradingview/charting_library.git
```
Comment (alnoki) [inline .gitmodules:6]: Is this relying on the private or public version?

If public, how about we keep it as `https` for accessibility?

(In multiple cases we've had downstream devs who aren't aware of SSH auth on GitHub and flag submodule issues)

Comment (xbtmatt) [reply .gitmodules:6]: That's the private version, and yes I agree, I'll change them both to `https`, is there any reason we have `inbox` as ssh?

Comment (alnoki) [reply .gitmodules:6]: So if it is the private version then unfortunately it'll have to use SSH

But we can use https for public repos

Please do go ahead and update it for inbox then

**Review by alnoki — APPROVED** [review]

File: src/inbox
Comment (alnoki) [inline src/inbox:1]: This should be updated in the inbox repo, not here

Then once the inbox PR lands we can bump the commit here

**Pull request merged**